exporters/otlp/otlp_exporter_benchmark.cc

```diff
@@ -0,0 +1,124 @@
+#include "otlp_exporter.h"
+
+#include <benchmark/benchmark.h>
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
```

Comment:
Usually there's no need for those namespaces in tests.

Reply:
I think it's need here to recognize `OtlpExporterTestPeer` as a friend class of `OtlpExporter`